lib/senders/email.js

```diff
@@ -911,7 +911,7 @@ module.exports = function (log, config) {
   Mailer.prototype.newDeviceLoginEmail = function (message) {
     log.trace({ op: 'mailer.newDeviceLoginEmail', email: message.email, uid: message.uid })
     var templateName = 'newDeviceLoginEmail'
-    var links = this._generateLinks(this.initiatePasswordChangeUrl, message.email, {}, templateName)
+    var links = this._generateLinks(this.accountSettingsUrl, message.email, {}, templateName)
```

Comment:
Woo new link!

Reply:
Why is `initiatePasswordChangeUrl` not here anymore?

Reply:
`this._generateLinks` first arg is actually the primary callout button link, which in this case we want it to be `accountSettingsUrl`.  The `initiatePasswordChangeUrl` is still created in the links so the user can change password.